musicround/__init__.py

```
@@ -177,148 +178,132 @@ def load_user(user_id):
+                    db.session.commit()
+                    app.logger.info(f"_app_update_token for Spotify (user {current_user.id}): Token successfully updated and committed to DB.")
                 except Exception as e:
-                    app.logger.error(f"Error getting client credentials token: {str(e)}")
-        
-        except Exception as e:
-            app.logger.error(f"Error in ensure_spotify_token: {str(e)}")
-            pass  # Continue without a token if all methods fail
+                    db.session.rollback()
+                    app.logger.error(f"_app_update_token for Spotify (user {current_user.id}): Error committing token to DB: {str(e)}", exc_info=True)
+            else:
+                app.logger.warning(f"_app_update_token for Spotify: Attempted to update token for unauthenticated user.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)